From 8e542e57adc1189e7bfd500b62441b3bc5c0ec1a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 27 May 2020 07:45:51 +0200
Subject: [PATCH 1/1] examples: do not build standalone applications with
 FTRACE

Compiling the standalone applications fails with FTRACE.
So skip them if FTRACE is defined.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 examples/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/examples/Makefile b/examples/Makefile
index d440bc5655..b8146d0c65 100644
--- a/examples/Makefile
+++ b/examples/Makefile
@@ -4,8 +4,8 @@ ifndef CONFIG_SANDBOX
 
 ifdef FTRACE
 subdir-ccflags-y += -finstrument-functions -DFTRACE
-endif
-
+else
 subdir-y += standalone
+endif
 subdir-$(CONFIG_API) += api
 endif
-- 
2.26.2

